### ISSUE TITLE
1.24.0 has dependency conflicts with rails 4.2.7.1

### ISSUE THREAD

**Issue by RobertoSchneiders** [Shopify/ssllabs.rb#27]
We really need `"json", '~> 2.0'` in the ssllabs gem?

The commit https://github.com/Shopify/ssllabs.rb/commit/231150c61652416b8e14d72e6158cb73ec91b46d introduced this and it breaks compatibility with the latest rails 4 available. 

```
Bundler could not find compatible versions for gem "json":
  In Gemfile:
    rails (= 4.2.7.1) was resolved to 4.2.7.1, which depends on
      activesupport (= 4.2.7.1) was resolved to 4.2.7.1, which depends on
        json (>= 1.7.7, ~> 1.7)

    ssllabs was resolved to 1.24.0, which depends on
      json (~> 2.0)
```

**Comment by EiNSTeiN-** [comment]
I think we could go for `>=1.7` since I don't think there is any specific incompatibility with any version.

**Comment by RobertoSchneiders** [comment]
Thank you @EiNSTeiN-. 👏

**Comment by EiNSTeiN-** [comment]
Released as [1.24.0.1](https://rubygems.org/gems/ssllabs/versions/1.24.0.1).
